libs/langchain/langchain/document_loaders/parsers/pdf.py

```diff
@@ -128,18 +128,29 @@ def _extract_images_from_page(self, page: pypdf._page.PageObject) -> str:
 class PDFMinerParser(BaseBlobParser):
     """Parse `PDF` using `PDFMiner`."""
 
-    def __init__(self, extract_images: bool = False):
+    def __init__(self, extract_images: bool = False, load_per_pages: bool = False):
```

Comment:
1) Could you document what this parameter does? Unlike the other parameter it's not obvious from the name what it does / controls etc.

Reply:
Let's do something like this

```python

    def __init__(self, extract_images: bool = False, *, concatenate_pages: bool = True):
        """Initialize a parser based on PDFMiner.

        Args:
            extract_images: Whether to extract images from PDF.
            concatenate_pages: If True, concatenate all PDF pages into one a single
                               document. Otherwise, return one document per page.
        """
        self.extract_images = extract_images
        self.concatenate_pages = concatenate_pages

```

Set the default to `True` otherwise it's a  breaking  change for existing users

Reply:
@eyurtsev I renamed the argument for PDFMinerParser and PDFMinerLoader. Thanks!